metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -47,7 +47,8 @@ Source2:        metron-elasticsearch-%{full_version}-archive.tar.gz
 Source3:        metron-data-management-%{full_version}-archive.tar.gz
 Source4:        metron-solr-%{full_version}-archive.tar.gz
 Source5:        metron-enrichment-%{full_version}-archive.tar.gz
-Source6:        metron-pcap-backend-%{full_version}-archive.tar.gz
+Source6:        metron-indexing-%{full_version}-archive.tar.gz
```

Comment:
Yeah, I believe it is - https://docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html-single/RPM_Guide/#id566895